tests/shared/src/test/scala/cats/tests/UnorderedFoldableSuite.scala

```diff
@@ -68,6 +68,14 @@ sealed abstract class UnorderedFoldableSuite[F[_]](name: String)(implicit
       assert(fa.count(Function.const(true)) === (fa.size))
     }
   }
+
+  test(s"UnorderedFoldable[$name].contains") {
+    forAll { (fa: F[String], v: String) =>
+      implicit val F: UnorderedFoldable[F] = instance
+      assert(fa.contains_(v) === (iterator(fa).toSet.contains(v)))
```

Comment:
A very minor one.. I think, since there's `iterator` available and handy, then we can even avoid `toSet`, but rather just call `iterator(fa).contains(v)` instead.

Despite you decide to adjust it or not, the PR looks really good to me now, thanks!